app/presenters/subscription_presenter.rb

```diff
@@ -33,10 +33,9 @@ def available_filter_hash
 
   def search_criteria_field(field, value)
     return if field.eql?("radius")
-    return if field.eql?("location_category")
     return if field.eql?("jobs_sort")
 
-    return render_location_filter(search_criteria["location_category"], value, search_criteria["radius"]) if field.eql?("location")
+    return render_location_filter(value, search_criteria_to_h["radius"]) if field.eql?("location")
```

Comment:
I think we got rid of the method `search_criteria_to_h`.

Reply:
Has been changed since :)